src/Makefile.am

```diff
@@ -292,7 +292,8 @@ noinst_HEADERS = \
 	detect-sip-stat-code.h \
 	detect-sip-stat-msg.h \
 	detect-sip-uri.h \
-	detect-smb-share.h \
+	detect-smb-filename.h \
+  detect-smb-share.h \
```

Comment:
nit: ident is off

Reply:
Done in https://github.com/OISF/suricata/pull/7048/commits/a83f189494b78192c1f3db055aac28c92c39f841